<ui version="4.0" >
 <class>GeneralSettingsDlg</class>
 <widget class="QWidget" name="GeneralSettingsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Allgemeine Einstellungen</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../simon.qrc" >:/images/icons/computer.svg</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="gbPassword" >
     <property name="title" >
      <string>Passwortschutz</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QCheckBox" name="cbCheckPassword" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>Aktivieren</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="lbPassword" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>Passwort:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lePassword" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbGeneral" >
     <property name="title" >
      <string>Allgemein</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="cbStartSimonOnBoot" >
        <property name="text" >
         <string>Starte simon bei Systemstart</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="cbStartJuliusAsNeeded" >
        <property name="text" >
         <string>Starte juliusd wenn n√∂tig</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="cbAutoConnect" >
        <property name="text" >
         <string>Beim simon-Start verbinden</string>
        </property>
        <property name="tristate" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QCheckBox" name="cbAskBeforeExit" >
        <property name="text" >
         <string>Nachfragen vor dem Beenden</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbPaths" >
     <property name="title" >
      <string>Pfade</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="lbCommands_3" >
          <property name="text" >
           <string>Kommandos:</string>
          </property>
          <property name="buddy" >
           <cstring>leCommands</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Programmkategorien:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Shortcuts:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbLexicon" >
          <property name="text" >
           <string>Lexikon:</string>
          </property>
          <property name="buddy" >
           <cstring>leLexicon</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbGrammar" >
          <property name="text" >
           <string>Grammatik:</string>
          </property>
          <property name="buddy" >
           <cstring>leGrammar</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbPrompts" >
          <property name="text" >
           <string>Prompts:</string>
          </property>
          <property name="buddy" >
           <cstring>lePrompts</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbVocab" >
          <property name="text" >
           <string>Vokabular:</string>
          </property>
          <property name="buddy" >
           <cstring>leVocab</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLineEdit" name="leCommands" >
          <property name="text" >
           <string>conf/commands.xml</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leProgramCategories" >
          <property name="text" >
           <string>conf/categories.xml</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leShortcuts" >
          <property name="text" >
           <string>conf/shortcuts.xml</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leLexicon" >
          <property name="text" >
           <string>model/lexicon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leGrammar" >
          <property name="text" >
           <string>model/model.grammar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePrompts" >
          <property name="text" >
           <string>model/prompts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leVocab" >
          <property name="text" >
           <string>model/model.voca</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../simon.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cbCheckPassword</sender>
   <signal>toggled(bool)</signal>
   <receiver>lePassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>99</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel" >
     <x>285</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbCheckPassword</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>126</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>62</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
